tb_db/crud.py

```diff
@@ -341,3 +343,128 @@ def create_miru_profiles(db: Session, miru_profiles_by_sample_id: dict[str, obje
         created_miru_profiles.append(db_miru_profile)
 
     return created_miru_profiles
+
+
+def get_miru_cluster_by_sample_id(db: Session, sample_id: str):
+    
+    query_result = db.query(Sample).filter(
+        Sample.sample_id == sample_id
+    )
+
+    miru_ids = []
+    for row in query_result:
+        miru_ids.append(utils.row2dict(row))
+    
+    for item in miru_ids:
+        if item['valid_until'] is None:
+            miru_cluster_id = item['miru_cluster_id']
```

Comment:
Could we clarify the variable names in this section? The items in the `query_result` are `Samples`. So when we iterate over them, convert them to dictionaries and store them in a new list, that list should probably be called `samples` or `sample_dicts`?